src/EFCore.PG/Storage/Internal/Mapping/NpgsqlArrayTypeMapping.cs

```diff
@@ -130,8 +143,10 @@ static bool Compare(TElem[] a, TElem[] b, ValueComparer<TElem> elementComparer)
                 // Note: the following currently boxes every element access because ValueComparer isn't really
                 // generic (see https://github.com/aspnet/EntityFrameworkCore/issues/11072)
                 for (var i = 0; i < a.Length; i++)
+                {
```

Comment:
I tend not to put curlies in this kind of case - the logic is very clear and does not need to be explicit...

Reply:
I don't mind omitting these braces, but I'm having trouble finding a way to configure ReSharper to recognize the style we're looking for.

I can configure ReSharper to not warn for omitted braces in one-liner `if` and `for` statements, but having a one-liner `if` _inside_ a one-liner `for` seems impossible. Do you have this configured on your end?

I don't mind omitting these braces, I just want to have a consistent ReSharper template. I think that would help encourage new contributors to feel more comfortable submitting PRs (and reduce the amount of nits we have to make).

More generally, my goal with these cleanups is to get to the point where solution-wide analysis is helpful again. Currently, turning on solution-wide analysis is super distracting since not all of our files follow a consistent style.